### ISSUE TITLE
Missing `'browser'` field of `package.json`

### ISSUE THREAD

**Issue by heejongahn** [vercel/swr#443]
When target is `web`, webpack use `['browser', 'module', 'main']`as default value for [`resolve.mainFields`](https://webpack.js.org/configuration/resolve/#resolvemainfields).

As `swr`'s `package.json` currently does not provide `'browser'` field, `'module'` entry point is used and thus the bundle breaks with syntax error in some old browsers. (e.g. IE11)

Sure, users can always override `resolve.mainFields` setting by themself, but wouldn't it would be nice if `swr` provided `"browser": "./dist/index.js",` so it works out of the box for old browsers? If core team agrees, I could come up with the PR (though it would be one-liner.)

**Comment by huozhi** [comment]
@heejongahn not sure which error did you receive but I think this issue https://github.com/vercel/swr/issues/278 is probably related. it would be good if you can provide more information about the error here for us to track.

usually the `"main"` field and `"module"` are pretty enough for bundler to consume, since they're targeting CommonJS and ESModule format of assets.

If you need to convert your application assets to other formats by webpack (like UMD or just IIFE), it's up to application layer. or you can also use babel to transform swr dist by babel to the format you need.

**Comment by huozhi** [comment]
for some reason I saw the ESModule build is using ESNext syntax 

`build:esm": "tsc --target ESNext --module ES6 --outDir esm`

@shuding is it ok to use ES5 or it's specified to ESNext to due to some special reason?

**Comment by heejongahn** [comment]
@huozhi  For me, it was `class` syntax in `esm/cache.js` which caused the error.

**Comment by huozhi** [comment]
I think for temporary work around you can include swr in the babel transform list, based on https://github.com/vercel/swr/issues/278

```js
{
  test: /\.(js|jsx)?$/,
  exclude: /node_modules\/(?!swr)/,
  use: {
    loader: 'babel-loader',
  },
},
```

**Comment by shuding** [comment]
@huozhi Yeah I think it's a mistake to use `ESNext`, totally okay to change it to `ES5`.

**Comment by shuding** [comment]
Fixed in `swr@0.2.3`.

**Comment by heejongahn** [comment]
Thanks!
